lib/components/app-switcher/AppSwitcher.vue

```diff
@@ -13,8 +13,8 @@
           :style="{ 'background-color': hover || menuOpen ? '' : currentAppColor }"
           v-on="on"
         >
-          <v-icon v-if="hasApps && (hover || menuOpen)" color="primary"> mdi-apps </v-icon>
-          <img v-else-if="currentApp" :src="base64ImageURI(currentApp.svg_icon)" />
+          <v-icon v-if="hasApps && (hover || menuOpen)" width="100%" color="primary"> mdi-apps </v-icon>
+          <img v-else-if="currentApp" width="100%" :src="base64ImageURI(currentApp.svg_icon)" />
```

Comment:
This is not valid HTML. The `width` attribute is defined as: `The intrinsic width of the image in pixels. Must be an integer without a unit.`.

Reply:
How about adding a `style="max-height: 50px; max-width: 50px"`?